node/src/GlideClusterClient.ts

```diff
@@ -567,4 +569,30 @@ export class GlideClusterClient extends BaseClient {
     public time(route?: Routes): Promise<ClusterResponse<[string, string]>> {
         return this.createWritePromise(createTime(), toProtobufRoute(route));
     }
+
+    /**
+     * Displays a piece of generative computer art and the server version.
+     *
+     * See https://valkey.io/commands/lolwut/ for more details.
+     *
+     * @param options - The LOLWUT options
```

Comment:
```suggestion
     * @param options - The LOLWUT options.
```
Please apply to other places too